lib/lf/uscalfe/lagr_fe.h

```diff
@@ -710,6 +710,455 @@ class FeLagrangePoint : public ScalarReferenceFiniteElement<SCALAR> {
   unsigned degree_;
 };
 
+/**
+ * @headerfile lf/uscalfe/uscalfe.h
+ * @brief Quadratic Lagrangian finite element on a triangular reference element
+ *
+ * This is a specialization of ScalarReferenceFiniteElement.
+ * Refer to its documentation.
```

Comment:
You can put a `@see ScalarReferenceFiniteElement` at the end of the comment if you want. That should generate a nice link :)